R/strategies.R

```diff
@@ -55,29 +66,53 @@ compute_sigma <- function(sigma_group, sigma_ref, index_mar){
 }
 
 
-#' Title
+
+#' Strategies
 #'
-#' @description
-#' TODO
+#' These functions are used to implement various reference based imputation
+#' strategies by combining a subjects own distribution with that of
+#' a reference distribution based upon which of their visits failed to meet
+#' the MAR assumption
+#'
+#' @param pars_group A list of parameters for the subject's group. See details.
+#'
+#' @param pars_ref A list of parameters for the subject's reference group. See details.
+#'
+#' @param index_mar A logical vector indicating which visits meet the MAR assumption
+#' for the subject. I.e. this identifies the observations that after a non-MAR
+#' intercurrent event (ICE).
+#'
+#' @details
+#'
+#' `pars_group` and `pars_ref` both must be a list containing elements `mu` and `sigma`.
+#' `mu` must be a numeric vector and `sigma` must be a square matrix symmetric covariance
+#' matrix with dimensions equal to the length of `mu` and `index_mar`. e.g.
+#' ```
+#' list(
+#'     mu = c(1,2,3),
+#'     sigma = matrix(c(1,2,3,4,5,6,7,8,9), nrow = 3, ncol = 3)
+#' )
+#' ```
+#'
+#' Users can define their own strategy functions and include them via the `strategies`
+#' argument to [impute()] using [getStrategies()]. That being said the following
+#' strategies are available "out the box":
+#'
+#' - Missing at Random (MAR)
+#' - Jump to Reference (JR)
+#' - Copy Reference (CR)
+#' - Copy Increments from Reference (CIR)
+#' - Last Mean Carried Forward (LMCF)
 #'
-#' @param pars_group TODO
-#' @param pars_ref TODO
-#' @param index_mar TODO
+#' @name stratgies
```

Comment:
I see you called it "stratgies" instead of "strategies". Is this what you want?

Reply:
typo. Fixed